flocker/node/agents/cinder.py

```diff
@@ -527,7 +527,7 @@ def attach_volume(self, blockdevice_id, attach_to):
             volume_manager=self.cinder_volume_manager,
             expected_volume=nova_volume,
             desired_state=u'in-use',
-            transient_states=(u'attaching',),
+            transient_states=(u'available', u'attaching',),
```

Comment:
We should update the `CinderAttachmentTests` to include this state. Currently they only include `attaching` as a possible transient state.